cmake/OatppWebSocketExternalProject.cmake

```diff
@@ -0,0 +1,24 @@
+#[[ CMake guard. ]]
+if(TARGET OatppWebsocketExternalProject)
+    return()
+endif()
+
+include(ExternalProject)
+include(${CMAKE_CURRENT_LIST_DIR}/OatppExternalProject.cmake)
+
+set(ROBOT_FARM_OATPP_WEBSOCKET_URL
+        "https://github.com/oatpp/oatpp-websocket/archive/1.0.0.zip"
+        CACHE STRING
+        "URL of the oatpp source archive")
+
+externalproject_add(OatppWebsocketExternalProject
+        PREFIX ${CMAKE_CURRENT_BINARY_DIR}/oatpp
```

Comment:
The directory on line 15 is required to be unique per external project. This will merge the source and build trees of oatpp and oatpp-websocket.